### PR TITLE
Officially support minio as a provider

### LINKED ISSUE #149: Support MinIO as storage provider
## 🚀 Feature Proposal

Add support for MinIO as storage provider.

## Motivation

We have a running MinIO instance and want to use it for our remote cache.
I tried to use it with the AWS S3 Storage Provider docs, but I can't get it working. 
So I think there are maybe some additional settings or steps needed.

## Example
Adding specific doc section for MinIO.

### PR DESCRIPTION
**Pull request by EWhite613**
## In this PR:

- Support MINIO as storage provider. Can just use s3 and `s3ForcePathStyle: true` and it'll work
  - fixes #149 in official manner (solutions already provided in comments on that issue)
## Issues reference:
 - https://github.com/ducktors/turborepo-remote-cache/issues/149

### Checklist:

* x ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/ducktors/turborepo-remote-cache/pulls) for the same update/change?

* [x] Have you lint your code with `pnpm lint` locally prior to submission?
* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your core changes, as applicable?
  *  **Any ideas on writing a good test?** The `s3` tests do `s3ForcePathStyle` for the s3 tests, and that's all Minio support requires in addition to s3. I could just copy the s3 tests and change the provider but it's gonna have the same output for that
* [x] Have you successfully ran build with `pnpm build` of your changes locally?
* [x] Have you successfully ran tests with `pnpm test` of your changes locally?
* [x] Have you commit using [Conventional Commits](https://github.com/ducktors/turborepo-remote-cache#how-to-commit)?

### CODE REVIEW

File: test/minio.ts [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    )
+
+    await t.test(
+      'should return 401 when wrong authorization token is provided',
+      async () => {
+        const response = await app.inject({
+          method: 'GET',
+          url: '/v8/artifacts/not-found',
+          headers: {
+            authorization: 'wrong token',
```
[the remark this answers is not in the document]
Comment (EWhite613) [reply test/minio.ts:75]: For tests, so I think this is fine

**Comment by matteovivona** [comment]
@all-contributors add @EWhite613 on code

**Review by fox1t — APPROVED** [review]
LGTM.
Thank you for this contribution! I think the tests are enough and we can land this!

**Pull request merged**